start.go

```diff
@@ -85,12 +85,17 @@ command(s) that get executed on start, edit the args parameter of the spec. See
 	},
 }
 
+func init() {
+	if len(os.Args) > 1 && os.Args[1] == "init" {
+		runtime.GOMAXPROCS(1)
```

Comment:
I was under the impression that GOMAXPROCS doesn't control goroutines spawned by the runtime:

>The GOMAXPROCS variable limits the number of operating system threads that can execute user-level Go code simultaneously. There is no limit to the number of threads that can be blocked in system calls on behalf of Go code; those do not count against the GOMAXPROCS limit. This package's GOMAXPROCS function queries and changes the limit.

This will still control goroutines spawned by packages, though do any of them do operations which require the thread locked?

In any case, it doesn't hurt to leave this for now (until it is deprecated), was just not sure it's strictly required.